[iir1] find_package failed

**Host Environment**
- OS: Windows
- Compiler: msvc2019

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install iir1`
`find_package(iir REQUIRED)`

**Failure logs**
```
[CMake] -- Found NumCpp: D:/CloneFromGithub/vcpkg/installed/x64-windows/include (found version 2.6.0)
1> [CMake] CMake Error at D:/CloneFromGithub/vcpkg/scripts/buildsystems/vcpkg.cmake:782 (_find_package):
1> [CMake]   Could not find a package configuration file provided by "iir" with any of
1> [CMake]   the following names:
1> [CMake] 
1> [CMake]     iirConfig.cmake
1> [CMake]     iir-config.cmake
1> [CMake] 
1> [CMake]   Add the installation prefix of "iir" to CMAKE_PREFIX_PATH or set "iir_DIR"
1> [CMake]   to a directory containing one of the above files.  If "iir" provides a
1> [CMake]   separate development package or SDK, be sure it has been installed.
1> [CMake] Call Stack (most recent call first):
1> [CMake]   src/CMakeLists.txt:15 (find_package)
1> [CMake] -- Configuring incomplete, errors occurred!
```

**Additional context**
the port's name is "iir1" while using ```find_package(iir)```:
![image](https://user-images.githubusercontent.com/40262194/161375832-c3d19322-471f-48ee-8aee-b544dd6ee618.png)

so the search path of ```iir-config.cmake```will be ```installed/x64-windows/share/iir```

however, vcpkg only creates dir ```installed/x64-windows/share/iir1```, thus```find_package(iir)``` failed

--- a/ports/iir1/portfile.cmake
+++ b/ports/iir1/portfile.cmake
@@ -12,7 +12,7 @@ vcpkg_cmake_configure(
 
 vcpkg_cmake_install()
 
-vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake)
+vcpkg_cmake_config_fixup(PACKAGE_NAME iir CONFIG_PATH lib/cmake/iir)
 
 vcpkg_copy_pdbs()
 

--- a/ports/iir1/vcpkg.json
+++ b/ports/iir1/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "iir1",
   "version": "1.9.0",
+  "port-version": 1,
   "description": "Realtime C++ filter library",
   "homepage": "https://github.com/berndporr/iir1",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2882,7 +2882,7 @@
     },
     "iir1": {
       "baseline": "1.9.0",
-      "port-version": 0
+      "port-version": 1
     },
     "ijg-libjpeg": {
       "baseline": "9d",

--- a/versions/i-/iir1.json
+++ b/versions/i-/iir1.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "393a498a50400c062dafdefbbbdbd3d1c3daef4a",
+      "version": "1.9.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "45098a249e45e71b1efa228efae07950b4499be4",
       "version": "1.9.0",
       "port-version": 0

